challenges/prototypes.js

```diff
@@ -1,33 +1,63 @@
 /* ===== Prototype Practice ===== */
 
-// Task: You are to build a cuboid maker that can return values for a cuboid's volume or surface area. Cuboids are similar to cubes but do not have even sides. Follow the steps in order to accomplish this challenge.
+// Task: You are to build a cuboid maker that can return values for a cuboid's 
+// volume or surface area. Cuboids are similar to cubes but do not have even sides.
+//  Follow the steps in order to accomplish this challenge.
+
 
 /* == Step 1: Base Constructor ==
-  Create a constructor function named CuboidMaker that accepts properties for length, width, and height
+  Create a constructor function named CuboidMaker that accepts properties 
+  for length, width, and height
 */
 
+function CuboidMaker(object) {
+  this.length = object.length;
+  this.width = object.width;
+  this.height = object.height;
+}
+
+CuboidMaker.prototype.volume = function volume(length, width, height) {
+  return this.length * this.width * this.height;
+};
+
+CuboidMaker.prototype.surfaceArea = function surfArea(length, width, length, height, width, height) {
```

Comment:
You don't need all those parameters. You should only have `width`, `length`, `height` stated once.